SalesforceCommerceCloud/pwa-kit
# Restart login flow if refresh_token is invalid

# Description



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

- rebuild commerce-react-sdk
- Run Retail App
- Go to localStorage, delete access_token
- Go to cookie, and change value in RefArch-cc-nx-g
- Refresh the page
- Observe that token returns an error, but the login flow started to fetch a new guest session

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
